src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,68 @@
+@import "../utils/mixins";
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @include square;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    cursor: pointer;
+    transition: background-color 0.5s;
+  }
+
+  &__day:hover {
+    transform: translateY(-20px);
+    background-color: pink;
```

Comment:
Place color in variable.

Reply:
Thank you for review of my code!

Reply:
> I`m using this color one time in this block, I think that I must not place this color in the variable.

In the future, you will have many pages where this color can be reused) I just tell you mentor remarks that I get))) 

You are welcome!)